packages/legend-query/src/stores/QueryBuilderState.ts

```diff
@@ -270,6 +282,39 @@ export class QueryBuilderState {
     }
   }
 
+  buildClassVersionValue(element: Class): void {
+    const versionPropertyParameter = new VariableExpression(
+      DEFAULT_VERSION_PARAMETER_NAME,
+      new Multiplicity(1, 1),
+      GenericTypeExplicitReference.create(
+        new GenericType(
+          this.queryParametersState.queryBuilderState.graphManagerState.graph.getPrimitiveType(
+            PRIMITIVE_TYPE.DATE,
+          ),
+        ),
+      ),
+    );
+    const parameterState = new QueryParameterState(
+      this.queryParametersState,
+      versionPropertyParameter,
+    );
+    parameterState.mockParameterValues();
+    this.queryParametersState.addParameter(parameterState);
+    const getAllFunction = buildGetAllFunction(
```

Comment:
why do we built this getAllFunction here? Do we use this anywhere?

Reply:
Don't we want to build the query when we try to `Execute...` from `versioned` class